sdcm/nemesis.py

```diff
@@ -1211,6 +1211,9 @@ def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
                 self.unset_current_running_nemesis(new_node)
         return new_node
 
+    def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
+        return self._nodetool_decommission()
```

Comment:
`return self._nodetool_decommission(add_node=add_node, disruption_name=disruption_name)`

Reply:
you are completely right, missed it in here.
fixed